index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello World</title>
+    <link rel="stylesheet" href="/style.css">
+</head>
+<body>
+    <h1>Hello Microverse</h1>
+</body>
```

Comment:
- [OPTIONAL] It is required to give a class name to your  `<h1>` tag and use that class name to style its content. This is marked as optional because it was not mentioned by the previous reviewer. But I highly recommend you to do that.

![project-rqts](https://user-images.githubusercontent.com/26597432/218732688-0dd5c887-7e4d-42b4-9bc3-127bef98094c.png)